exercises/concept/airport-robot/.meta/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "authors": [
+    "<your_gh_username>"
+  ],
+  "files": {
+    "solution": [
+      "src/lib.cairo"
+    ],
+    "test": [
+      "tests/airport_robot.cairo"
+    ],
+    "exemplar": [
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "go/airport-robot"
+  ],
+  "blurb": "<blurb>"
```

Comment:
If this is forked from Go, we can surely borrow its blurb.

Reply:
Will be implemented as part of the relevant ticket/PR, see https://github.com/exercism/cairo/pull/272#discussion_r1777017124